### DISCUSSION TITLE
How does the In Progress section work?

### DISCUSSION THREAD

**Discussion by SinTan1729** [IgnisDa/ryot#353] [General]
It seems to change on every refresh. Also, it sometimes shows completely finished series.
It would be great if either of these could be done:
1. Only add a series to In Progress if there are unseen episodes after the last seen one. (Preferably ignoring Specials or season 0.)
2. Make it completely manual.
3. Add an option to disable it.

**Comment by IgnisDa** [comment]
It's actually just the collection called in progress. I think you just imported from trakt. It can't really be determined if a show is in progress from import alone so ryot adds all of them to in progress. You can remove the item from the collection manually.
